jbpm-wb-process-runtime/jbpm-wb-process-runtime-api/src/main/java/org/jbpm/workbench/pr/model/ProcessSummary.java

```diff
@@ -23,72 +23,48 @@
 import org.jbpm.workbench.common.model.GenericSummary;
 
 @Portable
-public class ProcessSummary extends GenericSummary{
+public class ProcessSummary extends GenericSummary {
 
     private String processDefId;
     private String processDefName;
-    private String packageName;
-    private String type;
     private String version;
-    private String originalPath;
     private String deploymentId;
-    private String encodedProcessSource;
-
+    private boolean dynamic;
```

Comment:
I see that this property of a process definition in newly introduced in the model.
What does it denote exactly? What does it mean for process definition to be labeled as dynamic?